ompi/info/info.c

```diff
@@ -93,7 +80,7 @@ opal_pointer_array_t ompi_info_f_to_c_table = {{0}};
  * fortran to C translation table. It also fills in the values
  * for the MPI_INFO_GET_ENV object
  */
-int ompi_info_init(void)
+int ompi_mpiinfo_init(void)
```

Comment:
this is just a matter of taste ...
I would have spelled it ompi_mpi_info_init instead of ompi_mpiinfo_init, in order to make it more consistent with other parts of the code